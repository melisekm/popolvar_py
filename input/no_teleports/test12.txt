50 50 4700
HCCHHHCHCCHCHHCHHHHCCCCCHCHCCCCHCCHCCCCCHHHHHHCHCH
HCHCHCHCHCHCCHCHCHHCCHHCCHHHHCHHCCHHHCHCCHCHHCCCCC
HCCCHCCCHHHCCHCHCCCHHCHCCHCHCCCHCCHCHCCCHCHCHHCHHH
CHHCHHHCHHHHCCCHCHHCHCHHCHCHCHHCCHCCCHCCHHCHCCHCCH
CHCCHHCHCCHHCHHCHCCCCCCCHHCCCHCHCCHCHHCCHHHHCCHCHH
CHCHHCHCCHCCCCCCCCHCHHHCCHHCCHCHHHHCCCCCHHCHCCCHHH
HHCCCHCCHCCCHHHCCHHCCHCCHCHHCCHHHHHCHCHHCHHCCHCHHH
CCCHCHCHHHHHHCHCCCHCHCHHHHCHHCHCHHCHHHHCHCCCCCCHCH
CCCHHHCHCCHCHCCHCCHHCCCHCHHHCCHCCHCHCCHHHHCCHCHHCH
HHHHHCHHCCHHHCHHCCHHCCHHCCCHCHHCCHHCCCCHCCHHHCCHCH
HCHCHCCCHHHCCCCHHHCHCHCCHHCCCCCHHHCCCHCHCCHCHHHHHC
HCCHCHCHCCCHHHCCCCHCHCCHHCHHCHHCCCCHCCCHCHHHHCCPHH
HHCCHHCCCHHHHHCHCCHHHCCCCHCHHCHCCHHHHHHHCCCHHCCCHC
CCHCCHCCHHCHCHCHCHCCHHHCHCHCHHHCHHCCCHHCHCCCHCCCHH
CHCCCCHHHCCHCCCCHCCCHCCHCCCHCHCHCHCCCHCHHHHCCCCHHH
CCCCHCHHCCHCCHCHCCHCHHCHCHCCHHHCCHCCCHCHCHHCCCCCCC
CCHHCHCHCCHHCHHCCHCCCCCHHHCCCCHCCCHHHHHHHHHCHCHHCH
HHHCHCHCHHCHHHCHCHCHDCHHCHHCHHCHCCHCHHCHCCHHCHHCHC
HHCHCCCHCHHCHCCCHCCHCCCHCCCCHCCCHHCHCHCCCHCCHHHHHH
HHCHHHCHHHCHCHCCHHCCHHCCHHHCCCHCHCHCHHHHHHHHCHCHCC
CHHCHCHHCCCHHCHCHHHCHCHHCCCHHHCCCHCHCHHHCCHHCHCCHH
CCHHHHCHHHHHCCCHCHHCCCCCCHHHCHCHCHCHHCCCCHHHCHHHHC
CCCCCHHCHCCHCHHHCCHCCCCHCHCHHCHCCHCHCCCCCCHCHHHHCC
HHCHHHHCHHHCCCHCHHCCCHHCCCHHCCCCCCCCHHHHCCHHHHCHHH
HHHHHCCHHCHHCHCCHCHHHCHHCHCCHCHHCHHHCCCCCHCHCHHCCC
HCHCHHHCCCCHCCCHHHCCCCHHHCCHCCCHCHCHCHHHHCCCHCHCHC
HCHHCHHHHHHCCCCHCCHHCHCHHHHCCHCCCCHHHHHHCHCCCCHHHH
CHHHHCCCHCHCHHHHCHHCHCCHHHHHCHCCHCHCCCHHHHCHCHHHHH
CCHHCHHHHCCCCHCHHHHHCHHHCHHCCHCCHHHCCHHCCCHCHHCCHH
HCCCCHCCCHCCHCHHCHHHHHHHHHCCHCCCHCHCCHHHCHCHHCCCHC
CHHHCCHHHHHHHCHCHCHCCHCCCHCCCCHHCCHCCHHCCCHHCCCHCC
CHHHCCHCHHHCCHCHCHHCCHCCCCCHCHHCHHHCCCHHHHHHHHCHHH
CCHHCHCHCHHHCCCCHCHCHCCCHHCHCCCCCCCCHHHCCHHHCCHHHC
CHCCHCCCCCCHCHHCCCCHHCHHCHHCCCCCHCHHCHHCCCHHCHCCCH
CHCHCHCHHCCHHCCCCCCCHHCCCCCHHCHHCHCCCHCHHHHCHCCCCH
CHHHCCHHHCCCHCHHPCHCHCHHHHHHCHCCCHCCCHCCCCCCHHHHCC
HCHHCHHHCCCHHCHCCCCHCCHCHHCCHHHHHCCCHHCCCHCCHCHHCC
CCCHHHCHHCCCHCCCCCCHHHCHCCHCHCCCCHCCCHHCCCCCHCHCCC
CCCCCHHCCCHCHHCHCHHHHCHHHHHCCHHHCHCHCCHHHCHCCCCHCC
CCHCHCCHCCCHCCCHHCHCCCHCHHHHHCHHCHHHCHHHHCHHHCHHHH
HCHHHHCCHHCCCCCHHHCHCHHHCHCCCHHCHHCCHCCHHCHHHCHCCH
HHCHHCHCHHHHCHHCHHHCHHHHHHCCHHHCCHCCCHCHCHCHCCCCHH
HCCCHCHHCCCHCHHCCHHHCCCHHHCHHHHCCCCCCCCHHHHHHHCHHC
HHHHCCCHHHCCHHCHCCCHCHCHPHCHHCCHCCHCHHCHHHCHCCCHHH
HHCCHCHHCCCHCCHHHHHHHCHHHHHCHHCHHHHHHCCCCHCCCHHCCC
CCHCHHHHHCCCHHCHHCCCCHHCHCCHCCCCHHCHHHCHCCCCHHCCCC
CCHCCHHHCCHCCCCCCHHCHHHCCHCCHCHCCCCCHHCCHCCHCHHCHC
HHHHCHCCCCHHHHCCCHHHCHCHHCHCCHCHHHCCHCHCHHCHHCCHCH
CHHPCCCHHHHCHCCHCCHHHCHHCHCCHHCCHCHHHCHCHCHHHHHCHC
HCCHCCCHCHHHHCCCHHCHCCHHCHCHCHHCCHHHCHCCHCCCHHCHHH