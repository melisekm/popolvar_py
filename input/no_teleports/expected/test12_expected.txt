0 0
1 0
1 1
1 2
2 2
3 2
4 2
5 2
6 2
7 2
7 3
7 4
8 4
9 4
10 4
10 5
11 5
12 5
13 5
13 6
13 7
13 8
13 9
13 10
14 10
14 11
14 12
14 13
14 14
15 14
16 14
17 14
18 14
19 14
19 15
19 16
20 16
20 17
20 18
20 19
20 20
20 21
20 22
20 23
20 24
20 25
20 26
20 27
20 28
20 29
20 30
20 31
19 31
19 32
19 33
19 34
18 34
17 34
17 35
16 35
16 36
15 36
15 37
14 37
14 38
14 39
14 40
13 40
12 40
12 41
12 42
12 43
11 43
10 43
10 44
9 44
9 45
9 46
9 47
8 47
7 47
6 47
6 48
5 48
4 48
3 48
4 48
5 48
6 48
6 47
7 47
8 47
9 47
9 46
10 46
11 46
12 46
13 46
14 46
15 46
16 46
17 46
17 45
18 45
19 45
20 45
21 45
21 44
22 44
22 43
23 43
24 43
25 43
26 43
26 42
26 41
27 41
27 40
27 39
27 38
27 37
27 36
27 35
27 34
27 33
28 33
28 32
29 32
30 32
31 32
32 32
33 32
34 32
35 32
35 31
36 31
36 30
36 29
36 28
36 27
36 26
36 25
36 24
37 24
38 24
39 24
40 24
40 23
40 22
40 21
40 20
40 19
40 18
41 18
42 18
43 18
43 17
43 16
43 15
44 15
45 15
46 15
46 14
46 13
46 12
46 11
47 11